scripts/generate_pkcs7_tests.py

```diff
@@ -37,7 +37,7 @@ class TestData:
     Take in test_suite_pkcs7.data file.
     Allow for new tests to be added.
     """
-    mandatory_dep = "MBEDTLS_MD_CAN_SHA256"
+    mandatory_dep = "PSA_WANT_ALG_SHA_256"
```

Comment:
This looks right for 4.0, but doesn't it break 3.6? In the framework, we'll need to have dependencies that are conditional on the version.